scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -623,6 +623,8 @@
                                   ; don't need the functionality. But we need to set the file anyways
                                   ; to avoid a NPE.
                                   (.setFile kubeconfig (File. config-file))
+                                  (when kubeconfig-context
+                                    (.setContext kubeconfig kubeconfig-context))
```

Comment:
Why is this needed?

Reply:
In cases where you have a ~/.kube/config file that has multiple contexts, it won't know which context it should use and will pick one, which might not be the one you wanted it to use. With this change, we can now explicitly select which one.